vyper/parser/stmt.py

```diff
@@ -235,7 +251,10 @@ def call(self):
         )
         if isinstance(self.stmt.func, ast.Name):
             if self.stmt.func.id in stmt_dispatch_table:
-                return stmt_dispatch_table[self.stmt.func.id](self.stmt, self.context)
+                if self.stmt.func.id == 'reset':
```

Comment:
This seems interesting .. why not just put the implementation in the dispatch table?

Reply:
@charles-cooper Doesn't work without serious refactoring given that the `reset()` function self-mutates its input `stmt`, which is different than all the other functions in the `stmt_dispatch_table`.